iceoryx_hoofs/test/moduletests/test_posix_posix_call.cpp

```diff
@@ -291,6 +364,28 @@ TEST_F(PosixCall_test, IgnoringMultipleErrnosWhereOccurringErrnoIsLastInListSucc
     EXPECT_TRUE(internal::GetCapturedStderr().empty());
 }
 
+TEST_F(PosixCall_test, IgnoringErrnosByMultipleIgnoreErrnosCallsFails)
+{
+    internal::CaptureStderr();
+
+    constexpr int RETURN_VALUE = 17;
+    constexpr int ERRNO_VALUE = 18;
+
+    iox::posix::posixCall(testFunction)(RETURN_VALUE, ERRNO_VALUE)
+        .successReturnValue(1)
+        .ignoreErrnos(ERRNO_VALUE - 10)
+        .ignoreErrnos(ERRNO_VALUE + 13)
+        .ignoreErrnos(ERRNO_VALUE + 17)
+        .evaluate()
+        .and_then([&](auto&) { EXPECT_TRUE(false); })
+        .or_else([&](auto& r) {
+            EXPECT_THAT(r.value, Eq(RETURN_VALUE));
+            EXPECT_THAT(r.errnum, Eq(ERRNO_VALUE));
+        });
+
+    EXPECT_FALSE(internal::GetCapturedStderr().empty());
```

Comment:
Could you please add tests for suppressing the error message for certain errnos. Here you could use the `CaptureStderr` and check that it is empty or not.

Reply:
yes, I already mentioned it in https://github.com/eclipse-iceoryx/iceoryx/pull/807#discussion_r636466031 ;)

Reply:
done